remoting/remoting-bolt/src/main/java/com/alipay/sofa/rpc/message/bolt/AbstractInvokeCallback.java

```diff
@@ -60,6 +65,17 @@ protected AbstractInvokeCallback(ConsumerConfig consumerConfig, ProviderInfo pro
         this.request = request;
         this.context = context;
         this.classLoader = classLoader;
+        this.setRejectedExecutionPolicy(consumerConfig);
+    }
+
+    private void setRejectedExecutionPolicy(ConsumerConfig consumerConfig) {
+        if (null == consumerConfig || consumerConfig.getRejectedExecutionPolicy() == null) {
```

Comment:
If this is not one of these three strategies, the method will directly throw exceptions.